### ISSUE TITLE
right click doesn't work anymore

### ISSUE THREAD

**Issue by devvmh** [metamaps/metamaps#810]
gotta make right click work again - I need to inspect my elements!

**Comment by Connoropolous** [comment]
closed in https://github.com/metamaps/metamaps/commit/4f9b9460ad794261aa72fa950ce4a8080677f572
note: still won't be able to right click on a couple elements, for an intention reason which is that they are elements themselves which get brought up by right clicking, and if you're on windows, that also triggers the default right click menu to appear, undesirably. The elements are: the right click menu, the topic carousel, video bubbles, the canvas

**Comment by devvmh** [comment]
I want to test this more

**Comment by Bortseb** [comment]
Yea, right click intentionally doesn't work anymore.. It might suck a little for dev... but it made actually mapping for me pretty frustrating... and would definitely confuse anyone on a similar system to mine having two context menus pop up.

**Comment by devvmh** [comment]
yeah, i was hoping to find a way to make it more fine-grained. I didn't realize this event handler was the problem though, so probably it's very doable now
